#22231 - Add nulls-first parameter for custom field ordering (#22476)
netbox-community/netbox: docs/models/extras/customfield.md

If enabled, values from this field will be automatically pre-populated when cloning existing objects.

### Nulls First

> When ordering objects by this custom field, controls whether objects with no value (null) are sorted before (enabled, the default) or after (disabled) objects that have a value.

Reviewer: Minor suggestion to improve readability:
```suggestion
When ordering objects by this custom field, controls whether objects with no value (null) are sorted before or after objects that have a value. This option is enabled by default.
```
